contributed_definitions/nyaml/NXbias_spectroscopy.yaml

```diff
@@ -1,114 +1,92 @@
 category: base
 doc: |
-  Base classes definition for bias spectroscopy.
+  Base class definition for bias spectroscopy.
   
-  Bias sweeps while measuring arbitrary channels with I(V) curves.
+  The Scanning Tunneling Spectroscopy (STS), an extension of Scanning Tunneling Microscopy (STM),
+  is used to measure I(V) curves at each point of the surface. The dI/dV(V) can also be measured
+  with the help of numerical differentiation of the I(V) curve or from Lock-in amplifier.
```

Comment:
documentation does not explain why its name is bias

Reply:
Doc is updated.